plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -1701,8 +1706,110 @@ public void testTruncateDecimalTransform()
     @Test
     public void testBucketTransform()
     {
-        String select = "SELECT d_bucket, row_count, d.min AS d_min, d.max AS d_max, b.min AS b_min, b.max AS b_max FROM \"test_bucket_transform$partitions\"";
+        testBucketTransformForType(
+                "DATE",
+                ImmutableList.of(
+                        "DATE '2020-05-19'",
+                        "DATE '2020-06-19'",
+                        "DATE '2020-07-19'",
+                        "DATE '2020-08-19'"),
+                ImmutableList.of(1, 2, 3, 4),
```

Comment:
are `firstBucketValues` values meaningful? or can this be anything?

Reply:
they can be pretty much anything, it may even makes sense to remove them